### ISSUE TITLE
Missing one letter in mask

### ISSUE THREAD

**Issue by 7michaelg7** [airbnb/lottie-web#1689]
Hello,

I have this animation and everything looks perfect, but I cannot figure out why the 'T' is missing.

https://lottiefiles.com/share/3jq6pR

Thanks!

**Comment by Danjuanlab** [comment]
It seems to be a problem with the pre-comp mask, but according to your project file, all the supported functions are used. The strange thing is that I found that you only need to move the position of your T letter shape layer, so it can be show on demo webpage. But as for why there is such a bug, I don't know why.

**Comment by 7michaelg7** [comment]
> It seems to be a problem with the pre-comp mask, but according to your project file, all the supported functions are used. The strange thing is that I found that you only need to move the position of your T letter shape layer, so it can be show on demo webpage. But as for why there is such a bug, I don't know why.

Thanks, Danjuanlab. Just to confirm, you could not see a fix for this issue either?

**Comment by bodymovin** [comment]
@7michaelg7 yes, unfortunately subtracting masks try to calculate the surface of the needed mask to perform the masking, and sometimes if the layer is translated, it will miscalculate.
Any chance you can avoid subtracting masks for the animation?
You will get a better performance as weel if you use regular additive masks.

**Comment by 7michaelg7** [comment]
> @7michaelg7 yes, unfortunately subtracting masks try to calculate the surface of the needed mask to perform the masking, and sometimes if the layer is translated, it will miscalculate.
> Any chance you can avoid subtracting masks for the animation?
> You will get a better performance as weel if you use regular additive masks.

Good to know, thanks @bodymovin. Which subtraction masks are you referring to? Is an alpha matte a subtraction mask? This animation actually does not have any true masks.
